modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryUserRelLocalServiceImpl.java

```diff
@@ -138,6 +142,59 @@ public AccountEntryUserRel addAccountEntryUserRel(
 			accountEntryId, user.getUserId());
 	}
 
+	@Override
+	public AccountEntryUserRel addAccountEntryUserRelByEmailAddress(
+			long accountEntryId, String emailAddress, long[] roleIds,
```

Comment:
Change `roleIds` to `accountRoleIds` and make sure the assign roles method treats them as `accountRoleIds`

Reply:
Also update `AccountRoleLocalService` to handle bulk role assignment to a user